zcash_primitives/CHANGELOG.md

```diff
@@ -10,6 +10,51 @@ and this library adheres to Rust's notion of
 ### Added
 - Added in `zcash_primitives::zip32`
   - An implementation of `TryFrom<DiversifierIndex>` for `u32`
+- Added to `zcash_primitives::transaction::builder`
+  - `Error::InsufficientFunds`
+  - `Error::ChangeRequired`
+  - `Builder` state accessor methods:
+    - `Builder::params()`
+    - `Builder::target_height()`
+    - `Builder::transparent_inputs()`
+    - `Builder::transparent_outputs()`
+    - `Builder::sapling_inputs()`
+    - `Builder::sapling_outputs()`
+- `zcash_primitives::transaction::fees` a new module containing abstractions
+  and types related to fee calculations.
+  - `FeeRule` a trait that describes how to compute the fee required for a
+    transaction given inputs and outputs to the transaction.
+- Added to `zcash_primitives::transaction::components::sapling::builder`
+  - `SaplingInput` a trait that provides a minimized view of a Sapling input suitable
+    for use in change and fee computation.
+  - `SaplingOutput` a trait that provides a minimized view of a Sapling output suitable
+    for use in change and fee computation.
+  - `SaplingBuilder::inputs` and `SaplingBuilder::outputs`: accessors for Sapling
+    builder state.
+- Added to `zcash_primitives::transaction::components::transparent::builder`
+  - `TransparentInput` a trait that provides a minimized view of a transparent input suitable
+    for use in fee computation.
```

Comment:
This was renamed and moved to a new module; changelog needs updating. Also needs `OutputView` if both will be listed directly (but could also just say that the new modules were added for X purpose).